.changelog/unreleased/deprecated/1676-marker-supply-deprecation.md

```diff
@@ -0,0 +1 @@
+* deprecate: The `MarkerSupply` query has been deprecated. Please use the `SupplyOf` query from the `bank` module instead, which provides equivalent functionality [#1676](https://github.com/provenance-io/provenance/issues/1676).
```

Comment:
Please don't put context tags on changelog entries. I.e. Please remove the "deprecate: " part at the start. We know it's a deprecation because it's in that section, and also because it says so in the message.